### ISSUE TITLE
[textarea] Looks resizable but isn't.

### ISSUE THREAD

**Issue by erikkroes** [ing-bank/lion#165]
The textarea element shows a small icon in the bottom right corner in several browsers. This icon indicates that it should be resizable by dragging this icon. Except it can not be dragged. 
Suggestion to make it either resizable like it suggests, or make sure the icon doesn't show. I believe the second was the intention.
